plugins/mysql/mysql_disk_by_user

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor MySQL database disk usage per user
+#
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#   Add this to your /etc/munin/plugin-conf.d/munin-node:
+#       [mysql_disk_size]
+#       user mysql
+#       users_with_underscores foo_bar d_ritchie # usernames of users in the
```

Comment:
These need to be prefixed with `env.`, I guess?

Additionally: why is *underscore* relevant here? Does this need to be part of the variable name?

Reply:
Indeed, oops.

It's relevant (for some value thereof) because in our setup, the prefies (usernames in our case) sometimes may contain an underscore, while the directories are called "<username>_<dbname>". I know, it's a hack, I'll remove it if you prefer.

Reply:
I think, I was confused here, since you never explicitly mentioned, that *prefix* refers to the token before an *underscore*. Maybe you want to add this somewhere in the documentation.

Due to this specific meaning of the underscore character, the environment variable name looks fine to me.